### DISCUSSION TITLE
Duvida lógica list python

### DISCUSSION THREAD

**Discussion by Felipe-BT** [frontendbr/forum#1853] [Perguntas e Respostas]
Estou codificando um programa em Python onde ele entra 6 valores em uma lista, porém eu gostaria que após a lista estivesse completa ela reiniciasse entrando novos valores. Alguma ideia de como fazer isso?

**Comment by portothree** [comment]
Não ficou muito claro a sua dúvida, talvez possa redeclarar a lista como vazia para reincia-la, mas terá de ter certeza de já ter usado os valores antes se assim for útil para o seu programa.

De qualquer maneira este é o fórum de frontend, talvez possa ter ajuda de Python no [backend-br/forum](https://github.com/backend-br/forum).

**Comment by Felipe-BT** [comment]
Tipo dentro do while tem varias condições (if) caso uma delas não seja verdadeira eu quero reiniciar o while, pois o valor de entrada está sempre mudando.

**Comment by rapassos** [comment]
Acredito que a instrução que está buscando é o "continue", veja o link a seguir...
https://www.programiz.com/python-programming/break-continue
